package.json

```diff
@@ -38,6 +39,7 @@
     "test": "jest --runInBand",
     "test-bigquery": "MALLOY_DATABASE=bigquery jest --runInBand",
     "test-postgres": "MALLOY_DATABASE=postgres jest --runInBand",
+    "test-publisher": "MALLOY_DATABASE=publisher jest --runInBand",
```

Comment:
Is there a reason this is --runInBand?

Basically, unless you use a duckdb connection in your tests, you don't nee --runInBand